src/analysis/FVA/fluxVariability.m

```diff
@@ -220,47 +220,121 @@
 else
     PCT_status=0;  % Parallel Computing Toolbox not found.
 end
+minFlux = model.lb(ismember(model.rxns,rxnNameList));
+maxFlux = model.ub(ismember(model.rxns,rxnNameList));
+if ~minNorm
+    %We don't have to provide the solutions, so we can speed this up quiet
+    %a bit.
+    QuickProblem = LPproblem;
+    [Presence,Order] = ismember(rxnNameList,model.rxns);
+    QuickProblem.c(:) = 0;
+    QuickProblem.c(Order(Presence)) = 1;

```

Comment:
you are solving for a different problem here: max/min sum(v), which is different from FVA. Think about Ecoli growth and secretion of metabolites. 
btw min sum(v) is pFBA.

Reply:
Yes, and that is intentional.
The idea (which works quite well on larger models), is that a lot of fluxes are unrestricted, i.e. can reach their respective upper/lower bounds, and those are all determined by one single LP and don't need to be calculated again. 
I.e. (if you don't want individual solutions), you can simply solve this once, and get a quite large number of min and max values (most of those which are unconstraint), with 2 LPs instead of 2 per reaction LPs.

Reply:
Actually this is a cool idea.

Reply:
Thanks